src/parser/schwabJSONEACHistoryParser.ts

```diff
@@ -196,29 +196,23 @@ export function parseEACHistory(input: string): EAC.Transaction[] {
                     awardDate: parseDates(sellToCoverRowLine[FIELD_SELL_TO_COVER_AWARD_DATE])[0],
                 }
                 sellToCoverRows.push(sellToCoverRow);
-                i = i+2;
             }
             eacTransaction.rows = sellToCoverRows;
 
-            // Read 1 summary line
-            if (!_.isEqual(parsed.data[i+1], OPTIONS_DETAILS_HEADER)) {
-                throwParsingError(parsed.data[i+1], OPTIONS_DETAILS_HEADER);
-            }
-            const optionsDetailsLine = readLine(parsed.data[i+2], OPTIONS_DETAILS_HEADER);
+            const optionsDetailsLine = line.OptionExerciseSummary.Details;
             const optionsDetails = {
-                exerciseCostUSD: parseUSD(optionsDetailsLine[FIELD_OPTIONS_DETAILS_EXERCISE_COST]),
-                grossProceedsUSD: parseUSD(optionsDetailsLine[FIELD_OPTIONS_DETAILS_GROSS_PROCEEDS]),
-                netProceedsUSD: parseUSD(optionsDetailsLine[FIELD_OPTIONS_DETAILS_NET_PROCEEDS]),
+                exerciseCostUSD: parseUSD(optionsDetailsLine.ExerciseCost),
+                grossProceedsUSD: parseUSD(optionsDetailsLine.GrossProceeds),
+                netProceedsUSD: parseUSD(optionsDetailsLine.NetProceeds),
```

Comment:
The existing approach in the parsers has been to define all JSON field names as constants on the top of file instead of referencing them directly in the code. That way it's easier to avoid typoing the names